src/NServiceBus.AcceptanceTests/Encryption/EndpointTemplates/DefaultServer.cs

```diff
@@ -42,7 +42,8 @@ public BusConfiguration GetConfiguration(RunDescriptor runDescriptor, EndpointCo
             builder.CustomConfigurationSource(configSource);
             builder.EnableInstallers();
 
-            builder.DisableFeature<TimeoutManager>();
+            // TimeoutManager is currently required by Sagas
+            builder.EnableFeature<TimeoutManager>();
```

Comment:
yes we could. but that requires going through all tests and enabling it if required. Also as soon as we resolved the issue (see david's PR or when we remove timeout cancellation) we have to go through all tests again. I think it's easier for now to enable it for all tests as a temporary solution. I'm not sure if there is a significant performance impact (to the already very slow tests).